### ISSUE TITLE
Veryfy that schemas are generated properly

### ISSUE THREAD

**Issue by hashedone** [CosmWasm/sylvia#46]
Generated schemas should take the `#[serde(untagged)]` into account, and not introduce an additional message layer. Ensure it is a case, and if not we need to find a workaround for that.

**Comment by jawoznia** [comment]
Seems to be generated properly. @hashedone please take a look and if it's all fine then we can close this issue.

Currently our messages are generated as so:

```json
      "Cw1QueryMsg": {
        "oneOf": [
          {
            "type": "object",
            "required": [
              "can_execute"
            ],
            "properties": {
              "can_execute": {
                "type": "object",
                "required": [
                  "msg",
                  "sender"
                ],
                "properties": {
                  "msg": {
                    "$ref": "#/definitions/CosmosMsg_for_Empty"
                  },
                  "sender": {
                    "type": "string"
                  }
                }
              }
            },
            "additionalProperties": false
          }
        ]
      },
```

It corresponds to format generated for other cosmwasm messages like for exmaple:

 ```json
     "DistributionMsg": {
        "description": "The message types of the distribution module.\n\nSee https://github.com/cosmos/cosmos-sdk/blob/v0.42.4/proto/cosmos/distribution/v1beta1/tx.proto",
        "oneOf": [
          {
            "description": "This is translated to a [MsgSetWithdrawAddress](https://github.com/cosmos/cosmos-sdk/blob/v0.42.4/proto/cosmos/distribution/v1beta1/tx.proto#L29-L37). `delegator_address` is automatically filled with the current contract's address.",
            "type": "object",
            "required": [
              "set_withdraw_address"
            ],
            "properties": {
              "set_withdraw_address": {
                "type": "object",
                "required": [
                  "address"
                ],
                "properties": {
                  "address": {
                    "description": "The `withdraw_address`",
                    "type": "string"
                  }
                }
              }
            },
            "additionalProperties": false
          },
```

**Comment by hashedone** [comment]
> The thing to check on the first place is if flatten schemas are properly generated. There were no problems expected for interfaces, but we want to double check if contracts with own messages and at least one interface are ok too.

**Comment by hashedone** [comment]
0.2 is released - moving it to 0.3.

**Comment by hashedone** [comment]
Closed, it is working.
